metrics/prometheus_fake.go

```diff
@@ -524,6 +524,21 @@ func (p testSubcontainersInfoProvider) GetRequestedContainersInfo(string, v2.Req
 							TxQueued: 0,
 						},
 					},
+					DiskIo: info.DiskIoStats{
+						IoServiceBytes: []info.PerDiskStats{{
+							Device: "/dev/sdb",
+							Major:  8,
+							Minor:  0,
+							Stats: map[string]uint64{
+								"Async":   0,
+								"Discard": 0,
```

Comment:
Usually I set different values for different keys in tests to be sure that value from different keys are read correctly.